src/traces/cone/attributes.js

```diff
@@ -161,11 +160,13 @@ var attrs = {
     }
 };
 
-extendFlat(attrs, colorAttrs('', 'calc', true), {
-    showscale: colorscaleAttrs.showscale,
+extendFlat(attrs, colorscaleAttrs('', {
+    colorAttr: 'u/v/w norm',
```

Comment:
This give better attribute descriptions for cone `colorscale` and friends. For example, for `cmin` we have

> "Sets the lower bound of the color domain. Value should have the same units as the u/v/w norm and if set, `cmax` must be set as well."